ledger-api/grpc-definitions/com/daml/ledger/api/v1/transaction_filter.proto

```diff
@@ -56,6 +56,18 @@ message InclusiveFilters {
   // the available packages at the time of the query.
   // Optional
   repeated InterfaceFilter interface_filters = 2;
+
+  // A collection of templates for which the payload will be included in the
+  // ``create_arguments`` of a matching ``CreatedEvent``.
+  // SHOULD NOT contain duplicates.
+  // In contrast to the ``template_ids`` field, one can specify whether the blob should be provided
+  // alongside the create arguments.
+  // If a contract is simultaneously selected by a template filter and one or more interface filters,
+  // the corresponding ``include_create_arguments_blob`` are consolidated using an OR operation.
+  // All ``template_id`` inside need to be valid: corresponding template should be defined in one of
+  // the available packages at the time of the query.
+  // Optional
+  repeated TemplateFilter template_filters = 3;
```

Comment:
@mziolekda : wdyt of making the `template_ids` field deprecated?

cf. https://stackoverflow.com/questions/60490487/whats-the-best-way-to-deprecate-a-field-in-protocol-buffer-v3-reserved-vs-depre

Reply:
Good point. For the templates we should definitely follow the `deprecated_template_ids` approach variant. I would like to do the same to the

```
oneof arguments {
    // The contract arguments as typed Record
    Record deprecated_create_arguments = 3;
    // The contract arguments specified using an opaque blob extracted from the ``create_arguments_blob`` field
    // of a ``com.daml.ledger.api.v1.CreatedEvent``.
    google.protobuf.Any deprecated_create_arguments_blob = 5;
  }
``` 
WDYT?

Reply:
I would not change the names. I'd just add `[deprecated = true]` and let the downstream build tools (e.g., `javac` and `scalac`) handle the generation of the deprecation message.

Reply:
the reason why I like changing the names, because right now there was a clash between `arguments.create_arguments` and `create_arguments`, which has forced me to rename the new ones to `contract_create_arguments`. Not nice :-( I much rather have nice names for new fields and crummy names for old ones.

Reply:
See my comment here: https://github.com/digital-asset/daml/pull/17546#discussion_r1360396293

I believe we should give that a serious shot, and thereby also remove any need for introducing new fields.